cmd/kaf/produce.go

```diff
@@ -17,28 +19,57 @@ import (
 var (
 	keyFlag         string
 	headerFlag      []string
-	numFlag         int
+	repeatFlag      int
 	partitionerFlag string
 	timestampFlag   string
 	partitionFlag   int32
+	bufferSizeFlag  int
+	inputModeFlag   string
 )
 
 func init() {
 	rootCmd.AddCommand(produceCmd)
 
 	produceCmd.Flags().StringVarP(&keyFlag, "key", "k", "", "Key for the record. Currently only strings are supported.")
 	produceCmd.Flags().StringArrayVarP(&headerFlag, "header", "H", []string{}, "Header in format <key>:<value>. May be used multiple times to add more headers.")
-	produceCmd.Flags().IntVarP(&numFlag, "num", "n", 1, "Number of records to send.")
+	produceCmd.Flags().IntVarP(&repeatFlag, "repeat", "n", 1, "Repeat records to send.")
```

Comment:
good change, i  think it fits better than --num. i'm ready for some breaking changes in the next version :)